lib/l10n/intl_fr.arb

```diff
@@ -183,5 +183,8 @@
   "thank_you_for_the_feedback": "Merci pour le signalement",
 
   "flutter_license": "Flutter est la boîte à outils d'interface utilisateur de Google pour créer de superbes applications compilées en natif pour mobile, Web et bureau à partir d'une seule base de code. En savoir plus sur Flutter à ",
-  "flutter_website": "https://flutter.dev"
+  "flutter_website": "https://flutter.dev",
+
+  "not_found_title": "Page introuvable",
+  "not_found_message": "Oh oh... il semblerait que la page n'est pas accessible."
```

Comment:
Would be nice to show the page requested name ex: (la page [nom de la page] n'est pas accessible). Look at using a placeholder for that

Reply:
Yes could be good! I'll be on that